<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Copyright (C) 2009 Pascal Mues</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StereoViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Transferfunktion</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="uibtnTFOeffnen">
            <property name="text">
             <string>Öffnen</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Histogramm</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QComboBox" name="uicobHistogram">
            <item>
             <property name="text">
              <string>Linear</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Logarithmisch</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="uibtnTFSpeichern">
            <property name="text">
             <string>Speichern</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="TransferFunctionViewAlpha" name="uiTFAlpha">
          <property name="minimumSize">
           <size>
            <width>410</width>
            <height>110</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TransferFunctionViewColor" name="uiTFColor">
          <property name="minimumSize">
           <size>
            <width>410</width>
            <height>110</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Fenster</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>611</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Sammelfenster</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="uicobMultiWidgets">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="uibtnAddMultiWidget">
           <property name="text">
            <string>Sammelfenster hinzufügen</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="uibtnRemoveMultiWidget">
           <property name="text">
            <string>Sammelfenster schließen</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Fenster</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="uicobVolumeWidgets"/>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="uibtnAddVolumeWidget">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Fenster hinzufügen</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QPushButton" name="uibtnRemoveVolumeWidget">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Fenster schließen</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="6">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Fenster Einstellungen</string>
           </property>
           <widget class="QGroupBox" name="groupBox">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>51</y>
              <width>231</width>
              <height>111</height>
             </rect>
            </property>
            <property name="title">
             <string>Generellen Kameraeinstellungen</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Nearclipping Distanz</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="uidsbClipNear">
               <property name="minimum">
                <double>0.010000000000000</double>
               </property>
               <property name="maximum">
                <double>999.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>88.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Farclipping Distanz</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="uidsbClipFar">
               <property name="minimum">
                <double>1000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>20000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Blickfeld</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="uidsbFieldOfView">
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="minimum">
                <double>1.000000000000000</double>
               </property>
               <property name="maximum">
                <double>360.000000000000000</double>
               </property>
               <property name="value">
                <double>45.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>51</y>
              <width>231</width>
              <height>131</height>
             </rect>
            </property>
            <property name="title">
             <string>Stereoskopische Kameraeinstellungen</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Stereo Sicht ein/aus</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="uickbStereoAn">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Stereo Verfahren</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="uicobStereoVerfahren"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Stereobasis</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="uidsbStereobasis">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>1.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.002000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Fernpunktweite</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="uidsbFernpunktweite">
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>23</y>
              <width>141</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Zugeordnetes Sammelfenster</string>
            </property>
           </widget>
           <widget class="QComboBox" name="uicobMultiWidgetForWidget">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>20</y>
              <width>130</width>
              <height>22</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="uickbToggleFullScreen">
           <property name="text">
            <string>Vollbildanzeige</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionBilderstack_ffnen"/>
    <addaction name="actionVolumefile_ffnen"/>
    <addaction name="separator"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuBearbeiten">
    <property name="title">
     <string>Bearbeiten</string>
    </property>
    <addaction name="actionEinstellungen"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuBearbeiten"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBilderstack_ffnen">
   <property name="text">
    <string>Imagestack öffnen</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionEinstellungen">
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
  <action name="actionVolumefile_ffnen">
   <property name="text">
    <string>Volumefile öffnen</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TransferFunctionViewAlpha</class>
   <extends>QGraphicsView</extends>
   <header>_Volume/TransferFunction/TransferFunctionViewAlpha.h</header>
  </customwidget>
  <customwidget>
   <class>TransferFunctionViewColor</class>
   <extends>QGraphicsView</extends>
   <header>_Volume/TransferFunction/TransferFunctionViewColor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
